### ISSUE TITLE
tutorial is not working

### ISSUE THREAD

**Issue by bdidk235** [gedehari/IzzyEngine#25]
when i try to play tutorial this error appears:
```
hscript/Interp.hx (line 258)
hscript/Interp.hx (line 451)
izzy/gameplay/Stage.hx (line 41)
izzy/state/PlayState.hx (line 154)
flixel/FlxGame.hx (line 637)
flixel/FlxGame.hx (line 724)
flixel/FlxGame.hx (line 677)
flixel/FlxGame.hx (line 551)
openfl/events/EventDispatcher.hx (line 402)
openfl/display/DisplayObject.hx (line 1399)

Uncaught Error: [file_contents,data/characters/.json]
Please report this error to the GitHub page: https://github.com/gedehari/IzzyEngine
```

thank you

**Comment by flashintv** [comment]
by looking into this I think it just tries to load in a character that **doesnt exist**

**Comment by gedehari** [comment]
hmm I thought I got this fixed in 682b4cc73fe2a21b7eeb80eb23650af2b694f1a3 , what version are you using?

**Comment by bdidk235** [comment]
I used the latest version
